---
layout: default
---
# GB0178

## Graph-Break Type
*Short name describing what triggered the graph break*

constant fold exception

## Context
*Values or code snippet captured at the break point*

attempted to run function {fn} with arguments {args}

## Explanation
*Explanation of why the graph break was triggered*

Encountered exception when attempting to constant fold.

## Hints
*Hints on how to resolve the graph break*

- This is likely to be a Dynamo bug. Please report an issue to PyTorch.


## Additional Information

<!-- ADDITIONAL INFORMATION START - Add custom information below this line -->

<!-- ADDITIONAL INFORMATION END -->

[Back to Registry](../index.html)

[Click here to add Additional Info](https://github.com/pytorch-labs/compile-graph-break-site/edit/main/docs/gb/gb0178.md)
